package.json

```diff
@@ -268,7 +268,7 @@
     "@metamask/controller-utils": "^11.9.0",
     "@metamask/delegation-controller": "^0.4.0",
     "@metamask/delegation-deployments": "^0.11.0",
-    "@metamask/design-tokens": "^7.1.0",
+    "@metamask/design-tokens": "^8.0.0",
```

Comment:
Upgrading design tokens from [v7 to v8](https://github.com/MetaMask/metamask-design-system/blob/main/packages/design-tokens/CHANGELOG.md#800)